commands/create/nodepool/command.go

```diff
@@ -9,12 +9,13 @@ import (
 	"github.com/fatih/color"
 	"github.com/giantswarm/gscliauth/config"
 	"github.com/giantswarm/gsclientgen/models"
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
```

Comment:
This is very cool, but one of the instances in which we could have take a step back, and create a separate PR.